client/me/help/main.jsx

```diff
@@ -219,6 +224,51 @@ class Help extends React.PureComponent {
 		);
 	};
 
+	supportSessionCard = () => {
+		return (
+			<Card className="help__support-session-card">
+				<div className="help__support-session-text">
+					<h2 className="help__support-session-title">
+						{ this.props.translate( 'Schedule a support session' ) }
+					</h2>
+					<p className="help__support-session-description">
+						{ this.props.translate(
+							'Quick Start Support Sessions give you a way to talk to one of our Happiness Engineers via a screen share with audio.'
+						) }
+					</p>
+					<div className="help__support-session-actions">
+						<Button
+							className="help__support-session-action"
+							primary
+							href={ localizeUrl( 'https://wordpress.com/checkout/offer-quickstart-session/' ) }
+							onClick={ this.trackSupportSessionButtonClick() }
+						>
+							Schedule a session
+						</Button>
+						<Button
+							className="help__support-session-action is-link"
+							borderless
+							href={ localizeUrl( 'https://wordpress.com/support/quickstart-support/' ) }
+						>
+							Learn more
+						</Button>
+					</div>
+				</div>
+				<div className="help__support-session-illustration">
+					<img src={ supportSession } alt="" />
+				</div>
+			</Card>
+		);
+	};
+
+	trackSupportSessionButtonClick = () => {
+		//@TODO: What information do we want to track here?
+		const { isBusinessPlanUser } = this.props;
+		recordTracksEvent( 'calypso_help_support_session_card_click', {
+			is_business_plan_user: isBusinessPlanUser,
```

Comment:
Does this page show up for all users? If so, instead of tracking just if a user is a business plan user, we could track all types of users. That way, we have data about which demographic of users actually uses feature.

Reply:
I believe it does, yep! I'm not sure if we'll be showing this banner to all users or just folks who don't already have a support session scheduled.

Since this is an account-level page and not site-level, I'm not sure how we determine their plan, since there could be many different plans associated with one account.

I suppose we could set true/false flags for each plan type and include all of them if they have at least one of a type of plan?

Reply:
True, both good points. We could just grab the plan according to the user's primary site, but it's flailing if we don't have a real data request for which we're providing a solution. Cycling through any and all sites to extract plans when we don't know if they're wanted/needed seems overkill.

I think I'd just remove it myself (or do the primary site thing), until there's a clear requested case for specific data.